common/src/main/java/com/wynntils/gui/widgets/QuestButton.java

```diff
@@ -125,27 +118,8 @@ private void trackQuest() {
     }
 
     private void openQuestWiki() {
-        final String baseUrl = "https://wynncraft.fandom.com/wiki/";
-
-        // todo handle mini quest
-        String name = this.questInfo.getName();
-        String wikiQuestPageNameQuery = WebManager.getApiUrl("WikiQuestQuery");
-        String url = wikiQuestPageNameQuery + WebUtils.encodeForCargoQuery(name);
-        Request req = new RequestBuilder(url, "WikiQuestQuery")
-                .handleJsonArray(jsonOutput -> {
-                    String pageTitle = jsonOutput
-                            .get(0)
-                            .getAsJsonObject()
-                            .get("_pageTitle")
-                            .getAsString();
-                    Utils.openUrl(baseUrl + WebUtils.encodeForWikiTitle(pageTitle));
-                    return true;
-                })
-                .build();
-
-        RequestHandler handler = new RequestHandler();
-
-        handler.addAndDispatch(req, true);
+        McUtils.soundManager().play(SimpleSoundInstance.forUI(SoundEvents.EXPERIENCE_ORB_PICKUP, 1.0F));
```

Comment:
We should have a helper method that plays sounds, not really this pr however

Reply:
It's not that complicated, but I agree, a helper would be a good thing.

https://github.com/Wynntils/Artemis/issues/509